apps/api/src/app/order/order.service.ts

```diff
@@ -123,10 +123,29 @@ export class OrderService {
       data.SymbolProfile.connectOrCreate.create.assetSubClass = assetSubClass;
       data.SymbolProfile.connectOrCreate.create.dataSource = dataSource;
       data.SymbolProfile.connectOrCreate.create.name = name;
-      data.SymbolProfile.connectOrCreate.create.symbol = id;
+
+      let symbolProfileSymbol = id;
+
+      if (data.type === 'INTEREST') {
```

Comment:
Shouldn't it be the same behavior for all types according to https://github.com/ghostfolio/ghostfolio/issues/3381#issuecomment-2171894093?

Reply:
I wasn't sure if those comments meant "yeah, they go through the same code path when being created" or "yeah, we should reuse profiles for all those types". Given that I'm not too familiar with how non-INTEREST types are used across the codebase, I felt it was safer to start with just INTERESTs. Should we apply the same logic to all of them then?